docs/Manual.md

```diff
@@ -94,7 +94,8 @@ When invoking the compiler from the command line you can supply the following fl
 * ```--log-memory``` The executable will log all calls to malloc and free.
 * ```--optimize``` Removes safety checks (like array bounds access, etc.) and runs the C-compiler with the `-O3` flag.
 * ```--check``` Run the compiler without emitting any binary, just report all errors found (in a machine readable way).
-
+* ```--no-compile``` Don't run the compiler, just generate the C source.
+* 
```

Comment:
Nit: could we remove this trailing asterisk? It’s adding an empty bullet point.

Reply:
Yes